### PR TITLE
Update your_first_game.rst

### PR DESCRIPTION
**Pull request by robertsonb93**

### CODE REVIEW

**Comment by robertsonb93** [comment]
Added a missing step for getting the mob animations in the C# instructions.

File: getting_started/step_by_step/your_first_game.rst [hunk 1]
```diff
@@ -629,6 +629,7 @@ choose one of the three animation types:
 
     public override void _Ready()
     {
+        var _mobType = GetNode<AnimatedSprite>("AnimatedSprite").Frames.GetAnimationNames();
```
Comment (cbscribe) [inline getting_started/step_by_step/your_first_game.rst:632]: This should be `_mobTypes` to match with the next line.

Comment (robertsonb93) [reply getting_started/step_by_step/your_first_game.rst]: You're right!
I don't use copy and paste enough.

**Comment by cbscribe** [comment]
Thanks and congrats on your first contribution!

**Pull request merged**